connect/hwinfo.go

```diff
@@ -88,3 +89,42 @@ func uuid() (string, error) {
 	}
 	return string(output), nil
 }
+
+// getPrivateIPAddr returns the first private IP address on the host
+func getPrivateIPAddr() (string, error) {
+	output, err := execute([]string{"ip", "-4", "--oneline", "--brief", "address"}, false, nil)
```

Comment:
Why not use the built in `net.InterfaceAddrs()` like the ruby version did with `Socket.ip_address_list` ?

Reply:
> Why not use the built in `net.InterfaceAddrs()` like the ruby version did with `Socket.ip_address_list` ?

sure - that  way looks simpler. I will change it